kolla-ansible/trilio-datamover-api/Dockerfile_victoria_centos

```diff
@@ -18,6 +18,14 @@ RUN dnf config-manager --save --setopt=epel.skip_if_unavailable=true
 
 ##Install datamover packages
 ADD trilio.repo /etc/yum.repos.d/
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-*
```

Comment:
can you please add comment here why are we commenting these mirror list ?

Reply:
@shyam-biradar should we add comment in the dockerfile

Reply:
I have added the comment in the PR description now.